src/lib/registration/ui/base_system_registration_dialog.rb

```diff
@@ -328,41 +341,16 @@ def help_text
             "get updates and extensions.")
       end
 
-      # ask the user to confirm skipping the registration
+      # Show a warning about skipping the registration
+      #
       # @return [Boolean] true when skipping has been confirmed
-      def confirm_skipping
+      def show_skipping_warning
         # Popup question: confirm skipping the registration
-        confirmation = _("If you do not register your system we will not be able\n" \
+        warning = _("If you do not register your system we will not be able\n" \
             "to grant you access to the update repositories.\n\n" \
             "You can register after the installation or visit our\n" \
-            "Customer Center for online registration.\n\n" \
-            "Really skip the registration now?")
-
-        ret = Yast::Popup.YesNo(confirmation)
-        log.info "Skipping registration on user request" if ret
-
-        ret
-      end
-
-      # description text displayed in the main dialog (kind of help text)
-      # @return [String] translated description text
-      def info
-        # label text describing the registration (1/2)
-        # use \n to split to more lines if needed (use max. 76 chars/line)
-        info = _("Please select your preferred method of registration.")
-
-        if !Yast::Mode.normal
-          # add a paragraph separator
-          info += "\n"
-
-          # label text describing the registration (2/2),
-          # not displayed in installed system
-          # use \n to split to more lines if needed (use max. 76 chars/line)
-          info += _("If you skip product registration now, remember to register after\n" \
-              "installation has completed.")
-        end
-
-        info
+            "Customer Center for online registration.")
+        Yast::Popup.Message(warning)
```

Comment:
NP: A small inconsistency, I'd use `warning` variable and `Popup.Warning` method or `message` variable and `Popup.Message` method. With this code it's not obvious what is the intention. Is it a warning or an informative message?